docs/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md

```diff
@@ -0,0 +1,262 @@
+---
+title: Middlewares configuration - Strapi Developer Documentation
+description:
+---
+
+<!-- TODO: update SEO -->
+
+<!-- TODO: maybe decide between "options" and "sensible defaults" in wording -->
+# Middlewares configuration
+
+::: strapi Different types of middlewares
+
+In Strapi, 2 middleware concepts coexist:
+
+- **Strapi middlewares** are configured and enabled as global middlewares for the entire Strapi server application. The present documentation describes how to configure Strapi middlewares.<br/>Strapi also offers the ability to implement your own custom middlewares (see [middlewares customization documentation](/developer-docs/latest/development/backend-customization/middlewares.md)).
```

Comment:
Same here do we want to call them Strapi middlewares or just global?

Reply:
Discussed it with @alexandrebodin . The documentation team decided, for now, to still call them "Strapi middlewares", as "Strapi middlewares" can be global, API-related, or plugin-related.

This is to avoid confusion with route middlewares.

We can find a better wording if necessary for the release :-)